MantisOIDC.php

```diff
@@ -39,6 +39,9 @@ function config() {
 			'clientId'     => '',
 			'clientSecret' => '',
 			'redirect_uri' => '', # is set once the config page is saved
+			'oidc_scopes' => array('openid'),
+			'oidc_uid_claim' => 'name',
+			'oidc_find_usery_by' => 'name',
```

Comment:
looks like a typo here

```suggestion
			'oidc_find_user_by' => 'name',
```

Reply:
Thank you for finding this typo. **FIXED**